packages/apollo-server-core/src/requestPipelineAPI.ts

```diff
@@ -13,11 +13,14 @@ import {
 } from 'graphql';
 import { KeyValueCache } from 'apollo-server-caching';
 
+
 // TODO: Get FieldValue and ExecutionPatchResult from execute
 // Copying these types over from ./execute for now, because this compiles as a separate TypeScript
 // project it can't import these types from a relative file path or through /dist? There is
 // probably some config magic that needs to be done here to get this to work...
 
+/********************************* COPY START **********************************/
```

Comment:
what's this about?

Reply:
I just wanted to better denote what types were copied from the execute file.